JavaScript: Adding interactivity - added extra details to make content easy to follow (#38714)
mdn/content: files/en-us/learn_web_development/getting_started/your_first_website/adding_interactivity/index.md

> [!WARNING]
> If you haven't been following along with the rest of our course, [download this example code](https://codeload.github.com/mdn/beginner-html-site-styled/zip/refs/heads/gh-pages) and use it as a starting point.

> 1. Inside your `first-website` or `beginner-html-site-styled-gh-pages`(if you downloaded the above example code) folder, create a new folder named `scripts`.

Reviewer: ```suggestion
1. Inside your `first-website` folder or the example folder you have just downloaded, create a new folder named `scripts`.
```
